src/components/layout/InvestNav/index.tsx

```diff
@@ -108,6 +109,15 @@ export const InvestNav = () => {
         <div className={styles.nav_text_tutorial}>{t('Tutorial')}</div>
         <FontIcon name={FontIconName.External} size={16} />
       </NavLink>
+
+      <NavLink
+        to={{ pathname: RICOCHET_V1_LINK }}
+        className={styles.nav_link_desktop_only}
+        activeClassName={styles.nav_link_active}
+      >
+        <div className={styles.nav_text_tutorial}>{t('Ricochet V1')}</div>
```

Comment:
Before using `i18n`, we should declare it in [the config file](https://github.com/Ricochet-Exchange/ricochet-frontend/blob/8659efda50deb3f328d9c2f51a6ae6d2a9809be2/src/i18n/en/main.json) first.